### ISSUE TITLE
[Bug] Full screen navigation draw gesture interferes w/scrolling, non-whiteboard drawing, gestures. Make it optional + default off

### ISSUE THREAD

**Issue by don-reba** [ankidroid/Anki-Android#8936]
Since 2.15, I can't (pretend) draw on the screen anymore when reviewing kanji cards because any left-to-right swipe pulls out the side menu.

Update: Oh, turns out there is a "whiteboard" feature for this.

**Comment by mikehardy** [comment]
Yes! And you can save whiteboards and do all sorts of other things. We are still tracking a possible interaction between the new full-screen navigation drawer swipe and other features - for instance it may interfere with gestures in some areas, but during card review the whiteboard should work. If you see any other odd interactions let us know!

**Comment by Nanonej** [comment]
Hey there ! That's an issue for me for the same reason, but as the whiteboard may prevent it, I don't like to draw those pretended lines because I don't try to trace precisely the kanji but to get the line orders right.
So if an option to activate or disable it may be added in the settings I would be glad.

**Comment by mikehardy** [comment]
I had someone else mention in play store reviews today that it interferes with scrolling as well. It is a much bigger pain point than I thought.

I believe we need a new setting with it default to off, this is open for anyone to implement :pray:

**Comment by david-allison-1** [comment]
Refs (I believe): 
https://github.com/ankidroid/Anki-Android/issues/8070
https://github.com/ankidroid/Anki-Android/issues/8071 
https://github.com/ankidroid/Anki-Android/issues/8235 
https://github.com/ankidroid/Anki-Android/pull/8245

**Comment by ShridharGoel** [comment]
I'll work on this. We can add an option under the `Actions` section of `Gestures` in Settings.

**Comment by oakkitten** [comment]
while unconventional, i find that adding a small gesture exclusion zone (iirc you can do 200dp?) on the bottom of the screen works really well for doing both gesture navigation & opening the drawer. this is a simple fix that doesn't conflict with any other gestures

**Comment by shwnchpl** [comment]
I'm very happy to have a setting for this.  I had been having the exact issue described by the original reporter, but I figured my use case (pretending to draw Kanji on my phone screen) was too niche to justify a bug report.  I see that #9015 seems to have caused other issues so it has been reverted for the time being, but I just wanted to say that I appreciate the AnkiDroid maintainers/contributors for taking the time to fix this (and for making/maintaining an amazing piece of free software in general).
